lib/remote_coffee_slack/members.rb

```diff
@@ -19,19 +19,31 @@ def select_coffee_mates
 
     attr_reader :client, :members
 
-    def retrieve_all_members
-      all_members = []
-
-      client.users_list(presence: true, limit: 10, sleep_interval: 5, max_retries: 20) do |response|
-        all_members.concat(response.members)
+    def retrieve_channel_members
+      channel_members = []
+      params = {
+        channel: RemoteCoffeeSlack.config.slack_channel,
+        limit: 10,
+        sleep_interval: 5,
+        max_retries: 20
+      }
+
+      client.conversations_members(params) do |response|
+        channel_members.concat(response.members)
       end
 
-      all_members
+      channel_members
     end
 
     def members_handlers
-    members.map do |member|
-      "@#{member.id}|#{member.name}"
+      human_members.map do |member|
+        "@#{member}"
```

Comment:
Are you missing the id here?

Reply:
NVM, I thik we are only listing the members_id.

Just curious, did you test the user's tag in messages just using their id's?

Reply:
yep I did it. Sending only the id, slack shows the display name :D